commit 288503a9681adb908d8edb5af10b1c99557bf49f
Author: Joachim Wiberg <troglobit@gmail.com>
Date:   Sun Sep 6 11:16:22 2020 +0200

    Replace _PATH_VARDB with /var/lib/misc
    
    On many systems, e.g. uClibc, _PATH_VARDB still points to /var/db,
    which usually doesn't exist.   This is temporary workaround for a
    better fix that'll use RUNSTATEDIR instead.
    
    Signed-off-by: Joachim Wiberg <troglobit@gmail.com>

diff --git a/src/ssdpd.c b/src/ssdpd.c
index d6ddcf7..817bffc 100644
--- a/src/ssdpd.c
+++ b/src/ssdpd.c
@@ -384,7 +384,7 @@ static void lsb_exit(void)
 /* https://en.wikipedia.org/wiki/Universally_unique_identifier */
 static void uuidgen(void)
 {
-	const char *file = _PATH_VARDB PACKAGE_NAME ".cache";
+	const char *file = "/var/lib/misc/" PACKAGE_NAME ".cache";
 	char buf[42];
 	FILE *fp;
 
